packages/firebase/README.md

```diff
@@ -0,0 +1,68 @@
+<p align="center">
+   <br/>
+   <a href="https://next-auth.js.org" target="_blank"><img width="150px" src="https://next-auth.js.org/img/logo/logo-sm.png" /></a>
+   <h3 align="center">NextAuth.js - <b>Firebase Adapter</b></h3>
+   <p align="center">Authentication for Next.js</p>
+   <p align="center">
+   Open Source. Full Stack. Own Your Data.
+   </p>
+   <p align="center" style="align: center;">
+      <img src="https://github.com/nextauthjs/adapters/workflows/Build%20Test/badge.svg" alt="Build Test" />
+      <img src="https://github.com/nextauthjs/adapters/workflows/Integration%20Test/badge.svg" alt="Integration Test" />
+      <img src="https://img.shields.io/bundlephobia/minzip/@next-auth/firebase-adapter" alt="Bundle Size"/>
+      <img src="https://img.shields.io/npm/dm/@next-auth/firebase-adapter" alt="Downloads" />
+      <img src="https://img.shields.io/github/stars/nextauthjs/adapters" alt="Github Stars" />
+      <img src="https://img.shields.io/github/v/release/nextauthjs/adapters?include_prereleases" alt="Github Release" />
+   </p>
+</p>
+
+## Overview
+
+This is the Firebase Adapter for [`next-auth`](https://next-auth.js.org). This package can only be used in conjunction with the primary `next-auth` package. It is not a standalone package.
+
+## Getting Started
+
+1. Install `next-auth` and `@next-auth/firebase-adapter`
+```js
+npm install --save-prod next-auth @next-auth/firebase-adapter
+```
+
+2. Add this adapter to your `/src/pages/api/[...nextauth].js` next-auth configuration object.
+
+```js
+import NextAuth from "next-auth"
+import Providers from "next-auth/providers"
+import FirebaseAdapter from "@next-auth/firebase-adapter"
+
+// For more information on each option (and a full list of options) go to
+// https://next-auth.js.org/configuration/options
+export default NextAuth({
+  // https://next-auth.js.org/configuration/providers
+  providers: [
+    Providers.Google({
+      clientId: process.env.GOOGLE_ID,
+      clientSecret: process.env.GOOGLE_SECRET,
+    }),
+  ],
+  adapter: FirebaseAdapter.Adapter({
```

Comment:
```suggestion
import FirebaseAdapter from "@next-auth/firebase-adapter"

// For more information on each option (and a full list of options) go to
// https://next-auth.js.org/configuration/options
export default NextAuth({
  // https://next-auth.js.org/configuration/providers
  providers: [
    Providers.Google({
      clientId: process.env.GOOGLE_ID,
      clientSecret: process.env.GOOGLE_SECRET,
    }),
  ],
  adapter: FirebaseAdapter.Adapter({
```

The TypeORM one had a simple `Adapter` default import, while here it is `FirebaseAdapter.Adapter`. I think we should make it consistent. I am OK with either one or even simply a `FirebaseAdapter`, `TypeORMAdapter`  etc. default import should be fine. One module will only have a single adapter here anyway

Reply:
Yeah thats just how it was in the original implementation of his that I just copied into the docs for now. I agree we should keep it consistent and just export "Adapter" from them all though, or something similar :+1: